### PR TITLE
Show Confirm AlertDialog only when photos available to delete.

### PR DESCRIPTION
**Pull request by ananthrajsingh**
Refrain from showing Confirm delete AlertDialog when no photo is available. Show a toast for the same.

### CODE REVIEW

File: Camera2VideoJava/Application/src/main/java/com/example/android/camera2video/Camera2VideoFragment.java [hunk 1]
```diff
@@ -108,35 +109,70 @@
...
      * {@link TextureView}.
      */
-    private TextureView.SurfaceTextureListener mSurfaceTextureListener
+    private TextureView.SurfaceTextureListener mSurfaceTextureListener1
+            = new TextureView.SurfaceTextureListener() {
+
+        @Override
+        public void onSurfaceTextureAvailable(SurfaceTexture surfaceTexture,
+                                              int width, int height) {
+            Log.e(TAG, "onSurfaceTextureAvailable: TextureView width: " + width);
```
[the remark this answers is not in the document]
Comment (ananthrajsingh) [reply Camera2VideoJava/Application/src/main/java/com/example/android/camera2video/Camera2VideoFragment.java]: Hey @piotrek1543, you are right in saying so. Actually, I pushed some commits which were not meant to be pushed. I have rectified that.

File: Camera2VideoJava/Application/src/main/java/com/example/android/camera2video/Camera2VideoFragment.java [hunk 2]
```diff
@@ -268,19 +362,48 @@ private static Size chooseOptimalSize(Size[] choices, int width, int height, Siz
...
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+        Log.e(TAG, "onCreateView: ");
+        Handler handler = new Handler();
+        handler.postDelayed(new Runnable() {
+            @Override
+            public void run() {
+//                startRecordingVideo1();
+                startRecordingVideo2();
+                Toast.makeText(getActivity(), "Recording Started", Toast.LENGTH_LONG).show();
```
[the remark this answers is not in the document]
Comment (ananthrajsingh) [reply Camera2VideoJava/Application/src/main/java/com/example/android/camera2video/Camera2VideoFragment.java]: Advice taken, I'll keep in mind.

**Comment by owahltinez** [comment]
Thanks for this pull request! This makes sense, although I wonder if a better design would be to:

1. Disable the delete and share buttons when no media is present
2. Prevent users from going to the gallery if no photos are available
3. Take users back to camera when the last photo is deleted

Something worth thinking about is whether users can delete photos outside of the app. Photos are being saved in the app's files folder, so I don't think other apps can meddle with it but system apps (like the Files app) potentially can.

**Comment by owahltinez** [comment]
Closing due to inactivity

**Pull request closed without merging**